api/lib/firewall_functions.php

```diff
@@ -22,7 +22,9 @@
 
 require_once("/usr/libexec/nethserver/api/lib/Helpers.php");
 
-
+    if ($data['type'] === 'separator' ) {
+                success();
+    }
```

Comment:
The validation for separators is implemented  from line 134.
@stephdl should this code be removed?

Reply:
yes we could remove it, we just verify if the key exist or not, other are validated like any props of rules, good catch @gsanchietti

Reply:
Removed.